media-gfx/gnofract4d/gnofract4d-4.3_p20200821.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+DISTUTILS_SINGLE_IMPL=1
+
+inherit distutils-r1 eutils virtualx xdg
```

Comment:
optfeature is now an eclass by itself (see [here](https://archives.gentoo.org/gentoo-dev/message/50778e0b2e8b8b76a6612c5f9b7f1cec), recent change), so should be able to drop eutils.

Reply:
Changed to optfeature.